vendor/engines/split_accounts/spec/services/order_detail_splitter_spec.rb

```diff
@@ -13,8 +12,10 @@
     end
   end
 
+  let(:order) { build_stubbed(:order) }
+
   let(:order_detail) do
-    build_stubbed(:order_detail).tap do |order_detail|
+    build_stubbed(:order_detail, order: order).tap do |order_detail|
```

Comment:
I was actually talking about 
```ruby
build_stubbed(:order_detail, order: order, created_by: 1, quantity: 3, estimated_subsidy: BigDecimal("39.99"))
```
instead of the `tap`